core/client/pom.xml

```diff
@@ -188,6 +188,12 @@
           <scope>provided</scope>
         </dependency>
         <dependency>
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+          <version>3.0</version>
```

Comment:
what is the difference between specifying the version here vs specifying the version in the dependency management? don't we need to bump it up either way?